### ISSUE TITLE
Document Type Localization does not work in v14 and user defined language files

### ISSUE THREAD

**Issue by bassemelkadeem** [umbraco/Umbraco-CMS#16813]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.1.1

### Bug summary

when creating a localization file for document type and make a user defined language it does not reflect it backoffice

### Specifics

![1](https://github.com/user-attachments/assets/48f47d3a-0a9a-4176-9917-70394735816b)
![2](https://github.com/user-attachments/assets/0e896cd0-e631-4785-a2f2-9d08e2fb9124)
![3](https://github.com/user-attachments/assets/0d5d21cb-7679-4d38-8f3c-95689add19b5)
![4](https://github.com/user-attachments/assets/dad83b5c-b93f-48f3-81d0-13f1160ddb29)

### Steps to reproduce

follow the steps in the documentation link
https://docs.umbraco.com/umbraco-cms/fundamentals/data/defining-content/document-type-localization

### Expected result / actual result

expected result is that document types must appear in user defined language

---
_This item has been added to our backlog AB#42553_

**Comment by iOvergaard** [comment]
Hello, @bassemelkadeem. In V14 there are some things to be aware of to translate the Backoffice UI. Please have a look at this article in the documentation to register your language files: https://docs.umbraco.com/umbraco-cms/extending/language-files/ui-localization

That being said, you have indeed discovered a missing feature seeing, that we cannot currently replace localization keys on document types. That is something we intend to fix.

**Comment by bassemelkadeem** [comment]
Still the description for the property is not display in a correct way.

![image](https://github.com/user-attachments/assets/e3b43f27-4268-4d30-8365-38b9dfb605b2)
![image](https://github.com/user-attachments/assets/9bf50440-b273-45d6-b57d-9db5f3de1665)

**Comment by EddieDemon** [comment]
The manifest is not build completely, or at least it does not contain custom localizations. Localization is received by calling `manifest-[...].js` e.g. `/umbraco/backoffice/c50[...]/packages/core/en-[...].js` This file does not contain the custom translations.

This does not seem to be a BackOffice issue, unless these is a separate API that should have been called.

**Comment by leekelleher** [comment]
@bassemelkadeem Since [adding support for markdown](https://docs.umbraco.com/umbraco-cms/reference/umbraco-flavored-markdown) in property descriptions, the syntax to display localizations would be using the curly-brackets, e.g. `{#properties_pageTitleDesc}`.

**Comment by iOvergaard** [comment]
To extend on @leekelleher's comment, I have submitted a pull request to the docs to add a section explaining the link between adding custom localization keys and applying them to property descriptions: https://github.com/umbraco/UmbracoDocs/pull/6376

**Comment by EddieDemon** [comment]
> @bassemelkadeem Since [adding support for markdown](https://docs.umbraco.com/umbraco-cms/reference/umbraco-flavored-markdown) in property descriptions, the syntax to display localizations would be using the curly-brackets, e.g. `{#properties_pageTitleDesc}`.

It does not seem that that is what [the code](https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/src/libs/localization-api/localization.controller.ts#L183) is doing.

**Comment by EddieDemon** [comment]
@leekelleher I feel I'm not really following. The text is put through localization regardless of the brackets:

**With curly-bracket**
![afbeelding](https://github.com/user-attachments/assets/5c1aecb5-6c92-49de-88bb-a4a1bb9aeb8d)
![afbeelding](https://github.com/user-attachments/assets/59f31f83-5894-4709-9860-3eefe90a0437)

**Without curly-bracket**
![afbeelding](https://github.com/user-attachments/assets/cceb05b8-5908-4945-8eb4-6fc57e129a07)
![afbeelding](https://github.com/user-attachments/assets/cd1af26b-2761-4d7e-b919-f0087c69e2b2)

Both instances yield the input string as result.
![afbeelding](https://github.com/user-attachments/assets/8e4e912a-9338-4739-8eac-a9f22af525e2)

The UI Localization documentation that @iOvergaard changed mentions adding a plugin in which the localization is performed. This is another document than the author/I used to create the same(?) outcome.

**Comment by robgha01** [comment]
we are trying to do this with normal properties and property desctiptions before we could do this using #somekey and then create a dictionary for it to work but right now i feel like this functionality is removed and no longer works the umf only works in grid not in normal property places or am i wrong we cant get it to work and we did look at the documentation.

**Comment by iOvergaard** [comment]
@robgha01 You can render localization keys in the property title and description fields. The [Backoffice Localization article](https://docs.umbraco.com/umbraco-cms/customize-the-backoffice/foundation/localization) explains how to add custom localization keys or use existing ones in the code. At the bottom of the article, there is a section about "Using localization in property descriptions and labels". 

In summary, you add a localization key to the language files and render it in a property like this: `{#area_key}`. You can test it out with one of the built-in keys `{#general_ok}`.
